src/pathfora.js

```diff
@@ -2894,6 +2967,39 @@
 
     /**
      * @public
+     * @description public method to trigger widgets
+     * with the manualTrigger display condition
+     * @param {array}   widgetsIds (optional)
+     */
+    this.triggerWidgets = function (widgetIds) {
+      var pf = this;
+
+      // no widget ids provided, trigger all ready widgets
+      if (typeof widgetIds === 'undefined') {
+        pf.triggeredWidgets['*'] = true;
```

Comment:
No it doesn't get set back to false. Since it should trigger all widgets with the manualTrigger display condition, if a widget gets initialized after this function is called, we still need some kind of indicator that it should be triggered immediately. What are your thoughts on this?